packages/code/src/Code.tsx

```diff
@@ -146,6 +147,7 @@ function Code({
   language: languageProp,
   darkMode = false,
   showLineNumbers = false,
+  lineOffset = 0,
```

Comment:
Two notes on this:

1. Should the default here actually be `1`? We default this value to `1` in the rendering plugin, and since we're using a logical OR operator there, it's functionally defaulting this value to `1` anyways.
2. Given how this is used, we may want to rename this prop to something like `startingLineNumber` or `lineNumberStart`. Not only does that feel like a more intuitive API IMO, it also seems to be how we're using this value already.